openshift-client/src/main/java/io/fabric8/openshift/client/dsl/ImageSignatureOperation.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.openshift.client.dsl;
+
+import io.fabric8.kubernetes.client.dsl.CreateOrDeleteable;
+import io.fabric8.kubernetes.client.dsl.Nameable;
+import io.fabric8.openshift.api.model.ImageSignature;
+
+public interface ImageSignatureOperation extends Nameable<CreateOrDeleteable<ImageSignature>>, CreateOrDeleteable<ImageSignature> {
```

Comment:
This interface is specific to a single Resource, why can't we use something more generic?

Reply:
It can, I'll replace it with NameableCreateOrDeleteable.